manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -16,7 +16,16 @@ spec:
         name: tcp-istiod
       hosts:
         - "*"
+    {{- if or .Values.sidecarInjectorWebhook.injectionURL .Values.base.validationURL }}
+    - port:
+        number: 15053
+        protocol: TCP
+        name: tcp-istiodwebhook
```

Comment:
Well, this should be added for DNS-over-TLS, but please add a separate one for webhook, on 15017.

Reply:
changed to use 15017 for webhook